### PR TITLE
autopilot_tester: fix speed factor to fix CI

### PR DESCRIPTION
**Pull request by sfuhrer**
### Solved Problem
MAVSDK VTOL standard wind ramp test failing

### Solution
- increase the timeout
- reduce flown distance
- take speed factor into account for timeout

### Changelog Entry
For release notes:
```
Bugfix: CI MAVSKD testes: fix VTOL wind ramp test failing due to timeout
```

There's also other timeouts in autopilot_tester that are meant to be in simulation time but are currently in real time, though the mission end timeout seemed to have been the only one causing issues (prob because the timeouts are set more constraining there then on others that are shorter).

### CODE REVIEW

**Comment by sfuhrer** [comment]
Converted to draft because it's still failing in CI, though not locally. I'm on it.

**Comment by sfuhrer** [comment]
This indeed seems to make the wind SITL test more robust. The other 4 CI failures that happen in this specific PR atm are not related to it.

**Comment by sfuhrer** [comment]
@julianoes any chance you could review?

**Review by bkueng — APPROVED** [review]

File: .github/workflows/sitl_tests.yml [hunk 1]
```diff
@@ -96,7 +96,7 @@ jobs:
         PX4_HOME_LON: ${{matrix.config.longitude}}
         PX4_HOME_ALT: ${{matrix.config.altitude}}
         PX4_CMAKE_BUILD_TYPE: ${{matrix.config.build_type}}
-      run: test/mavsdk_tests/mavsdk_test_runner.py --speed-factor 20 --abort-early --model ${{matrix.config.model}} --upload test/mavsdk_tests/configs/sitl.json --verbose
+      run: test/mavsdk_tests/mavsdk_test_runner.py --speed-factor 3 --abort-early --model ${{matrix.config.model}} --upload test/mavsdk_tests/configs/sitl.json --verbose
```
Comment (MaEtUgR) [inline .github/workflows/sitl_tests.yml:99]: Doesn't it also work with speed factor 20 now?

**Review by MaEtUgR — APPROVED** [review]

File: .github/workflows/sitl_tests.yml (hunk 1 above)
Comment (sfuhrer) [reply .github/workflows/sitl_tests.yml]: Let's see, I've removed this commit for now.

Comment (sfuhrer) [reply .github/workflows/sitl_tests.yml]: seems to be okay as well without it

**Review by julianoes — COMMENTED** [review]
Sorry, I'm too late here. Looks good, thanks all!

**Pull request merged**